src/Uno.UI/UI/Xaml/Controls/ContentDialog/ContentDialog.cs

```diff
@@ -114,10 +149,50 @@ protected override void OnApplyTemplate()
 		{
 			base.OnApplyTemplate();
 
-			UpdateButtonsVisualStates();
-			ApplyDefaultButtonChanged(DefaultButton);
+			GetTemplateParts();
+
+			m_dialogMinHeight = ResourceResolver.ResolveTopLevelResourceDouble("ContentDialogMinHeight");
+
+			// For dialogs that were shown when not in the visual tree, since we couldn't prepare
+			// their content during the ShowAsync() call, do it now that it's loaded.
+			if (m_placementMode == PlacementMode.EntireControlInPopup)
+			{
+				PrepareContent();
+			}
+
+			// UNO TODO
+			//IFC_RETURN(m_epLayoutRootGotFocusHandler.AttachEventHandler(
+			//m_tpLayoutRootPart.AsOrNull<IUIElement>().Get(),
+
+			//[this](IInspectable *, xaml::IRoutedEventArgs *)
+
+			//{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6389897)